src/BinSkim.Sdk/CompilerDataLogger.cs

```diff
@@ -387,7 +348,6 @@ internal void SendChunkedContent(string eventName, string contentId, string cont
 
                 this.telemetryClient.TrackEvent(eventName, properties: new Dictionary<string, string>
                 {
-                    { "sessionId", this.sessionId },
```

Comment:
>sessionId [](http://example.com/codeflow?start=23&length=9)

Roughly related to previous comment, how should we be tracking version data in our telemetry? If BinSkim 1.0 has a bug in the source link rule, and we ship a fix in BinSkim 1.1, we'll want to assess behavior of the most current release (and also measure how many teams need to upgrade from 1.0 to the fixed version). How best to enable this?

Reply:
You found where we set the version in the global context so it's attached to all telemetry items.